environments/jvm/pom.xml

```diff
@@ -43,7 +43,7 @@
 	<repositories>
 		<repository>
 			<id>fission-java-core</id>
-			<name>fission-java-core-snapshot</name>
+			<name>fission-java-spring-snapshot</name>
```

Comment:
In order to support backward compatibility - we should let `fission-java-core` be as it is